ENIGMAsystem/SHELL/Graphics_Systems/None/NONEModelStruct.h

```diff
@@ -0,0 +1,465 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
```

Comment:
This file's another great example of resource management logic that we could move into a shared directory and extend in each system... don't worry about it for now, though.

Reply:
I will get that one because we still need a good vertex_buffer/index_buffer abstraction, and the generic model struct should make use of those.